.github/workflows/bcr/source.template.json

```diff
@@ -0,0 +1,5 @@
+{
+    "integrity": "**leave this alone**",
```

Comment:
do we actually have to have exactly this content in the value?

Reply:
Nope